gen/generate.go

```diff
@@ -105,6 +105,10 @@ func Generate(m *compile.Module, o *Options) error {
 	genBuilder := newGenerateServiceBuilder(importer)
 
 	generate := func(m *compile.Module) error {
+		if err := duplicateAfterNormalization(m.ThriftPath); err != nil {
```

Comment:
I think that this error check may make more sense in the `generateModule` function, since that is where we attempt to normalize the file name.